hubstorage/resourcetype.py

```diff
@@ -26,7 +28,9 @@ def _iter_lines(self, _path, **kwargs):
 
         r = self.client.request(**kwargs)
 
-        return r.iter_lines()
+        lines = r.iter_lines()
+        return (l.decode('utf8') if isinstance(l, six.binary_type)
+                else l for l in lines)
```

Comment:
> the response coming from Hubstorage has no encoding set

how so? `Content-Type: application/x-jsonlines; charset=UTF-8`

> I couldn't reproduce the issue you had.

I deleted the branch and pulled it again - the issue persists, please send me the whole output (py3 test, change branch, master output) in private message